jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -95,6 +114,58 @@ def generate_paper_letters(self):
             self._app.logger.error(f'Error generating batch letters: {e}')
         self._app.logger.debug('Finish generating batch letters.')
 
+    def upload_to_sftp(self, client, data, filename):
+        """SFTP data to targeted destination."""
+        return client.putfo(
+            fl=BytesIO(data),
+            remotepath=f'{self._app.config.get("BCMAIL_SFTP_STORAGE_DIRECTORY")}/{filename}'
+        )
+
+    def update_notes_and_status(self, furnishings_list, funishing_status, furnishing_notes=None):
+        """Update the notes and status of furnishing entries in a list."""
+        for furnishing in furnishings_list:
+            furnishing.notes = furnishing_notes
+            furnishing.status = funishing_status
+            furnishing.save()
+
+    def process_paper_letters(self):
+        """Process the generated paper letts of BC and XPRO businesses (SFTP)."""
+        # Skip SFTPing PDF files to BCMail+ if flag is on
+        if self._disable_bcmail_sftp:
+            self._app.logger.debug(f'disable-dissolution-sftp-bcmail flag on: {self._disable_bcmail_sftp}')
+            return
+
+        current_date = datetime_util.now()
+        if self._bc_mail_furnishings:
+            try:
+                filename = current_date.strftime('DIS1_LETTER_BC_%b.%d.%Y.pdf')
```

Comment:
The filename is different depending on if BC batch letter or XPRO batch letter (from the MIRO board).